.github/workflows/sidesail.yml

```diff
@@ -42,9 +42,11 @@ jobs:
       # You can specify other versions if desired, see documentation here:
       # https://github.com/dart-lang/setup-dart/blob/main/README.md
       - uses: dart-lang/setup-dart@v1
-
       - name: Install dependencies
-        run: flutter pub get
+        run: |
+          cd sidesail
+          flutter pub get
+          cd ..
```

Comment:
`cd ..` does nothing here, as working directories are not persisted through tasks. And would be better with this instead IMO

```
run: flutter pub get
working-directory: sidesail
```

Reply:
thanks